app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -169,15 +209,38 @@ fun HomeScreen(
       selectedTabIndex = 0
     }
 
+      if(uiState.isFilterDialogOpen) {
```

Comment:
It doesn't look like this ever becomes true. I tested on the emulator and the filter pop up does not show after clicking it.

Reply:
Sorry - I forgot to push the latest version. This will be updated in a separate pull request to implement functionality via the VM.